docs/src/atomistic/outputs.rst

```diff
@@ -167,3 +167,43 @@ Energy ensemble gradients
 
 The gradient metadata for energy ensemble is the same as for the ``energy``
 output (see `Energy gradients`_).
+
+.. _features:
+
+Features
+^^^^^^^^
+
+Last layer features represent the outputs generated by the model before the final predictions,
+such as energy or forces, are made. They are associated with the ``"features"`` key in the model
+outputs, and must adhere to the following metadata:
+
+.. list-table:: Metadata for features output
+  :widths: 2 3 7
+  :header-rows: 1
+
+  * - Metadata
+    - Names
+    - Description
+
+  * - keys
+    - same as `Energy`_
+    - same as `Energy`_
+
+  * - samples
+    - same as `Energy`_
+    - same as `Energy`_
+
+  * - components
+    - same as `Energy`_
+    - same as `Energy`_
```

Comment:
I think it is worth repeating the information here, we used same as `Energy` for the energy ensemble which is related, but this output is not related to the energy and might end up in a different page.

Reply:
So, you think we need to split the pages with the standard outputs related and not related to the energy?

Reply:
We don't need to do it now, but we will likely do something like this at some point. For now just having the documentation for this output be self contained should be enough!

Reply:
Ok ok, I added the doc